java/src/jmri/jmrix/loconet/messageinterp/MessageInterpretation.properties

```diff
@@ -1553,3 +1553,7 @@ LN_MSG_OPC_EXP_SPECIALSTATUS_FLAGS = Rsync Max {0} USB Connected {1} Other Flags
 LN_MSG_ON  = On
 LN_MSG_OFF = Off
 LN_MSG_COMMAND_STATION = (CS)
+
+LN_MSG_EXTEND_ACCY_CV_WRITE = Extended Accessory Decoder CV Write: Address {0} CV {1} with {2}.\n
+LN_MSG_EXTEND_ACCY_CV_VERIFY  = Extended Accessory Decoder CV Verify: Address {0} CV {1}, check if it is {2}.\n
+LN_MSG_EXTEND_ACCY_CV_BIT_ACCESS  = Extended Accessory Decoder CV Bit {} bit, Address {}, CV {}, bit # {} (of bits 0-7) with value {}.\n
```

Comment:
Static analysis fails on this line. The numbers between { and } are missing.

Reply:
addrssed, including logic improvements and tests